app/src/main/java/com/example/henzoshimada/feeltrip/Mood.java

```diff
@@ -39,26 +47,25 @@
     3 ----> socialSitChanged;
     4 ----> isPrivateChanged;
     5 ----> imageChanged;
-    6 ----> geoLocationChanged;
+    6 ----> latitudeChanged;
+    7 ----> longitudeChanged;
     */
 
-    private ArrayList<Boolean> stateVector;
```

Comment:
An ArrayList of Booleans can't be as easily filled/read as a boolean array can. A very minor optimization.